jdaviz/configs/cubeviz/cubeviz.yaml

```diff
@@ -32,6 +32,7 @@ tray:
   - cubeviz-moment-maps
   - cubeviz-spectral-extraction
   - imviz-aper-phot-simple
+  - g-data-quality
```

Comment:
I know we don't have a "policy" for this [:cat:](https://jira.stsci.edu/browse/JDAT-2104), but I think it might make sense to have this higher in the list.  I personally think of the plugin tray as order in typical workflow order - with metadata and visualization options at the top, more advance workflow after, and exporting at the bottom.  Under that scheme, this probably would make more sense somewhere after metadata and before gaussian smooth (and similar for the Imviz one).

Reply:
I wanted to chat about this too. I don't have a strong opinion on where it goes. 

I did want to ask if we want the DQ visible by default, and if so, in which configs. If you're loading an L2 product and specifying a DQ extension in Imviz, you probably want it open. But in Cubeviz, there's only one flag used by `jwst` and it's minimally informative. So I wouldn't be opposed to setting the DQ layer to not-visible by default in Cubeviz. Thoughts?

Reply:
I don't have strong thoughts on that right now - we might have to try it and get feedback.  The default "red" mask is a bit of a jarring change that probably isn't terribly helpful information when loading data, so I think I'd be supportive of either: not loading the DQ layer by default (but then its difficult to load it later), hide the layer by default, or choose a less intrusive default color.

Reply:
I am leaning towards "load but hide by default". With the new data panel Jenn is designing, we might get an easy access to showing DQ without having to wander in the plugin tray. Happy to be convinced otherwise.

Reply:
Done in 2e50d2e.

Reply:
In 08bed54 I've put DQ below Plot Options (higher up in the tray order).